src/test/scala/cromwell/engine/store/SymbolStoreSpec.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.engine.store
+
+import cromwell.engine.store.SymbolStore.SymbolStoreKey
+import org.scalatest.{FlatSpec, Matchers}
+import cromwell.binding.{WdlExpression, WdlNamespace}
+import cromwell.binding.types.{WdlExpressionType, WdlIntegerType, WdlStringType}
+import cromwell.binding.values.{WdlValue, WdlInteger, WdlString}
+import SymbolStoreSpec._
+
+object SymbolStoreSpec {
+  val Wdl =
+    """
+      |task a {
+      |  command { echo '${message}' }
+      |  output {
+      |    String message = read_string(stdout())
+      |    Int constant = 100
+      |  }
+      |}
+      |task b {
+      |  command { echo '${message} - ${Int integer}' }
+      |}
+      |workflow wf {
+      |  call a
+      |  call b {
+      |    input: message=a.message, integer=a.constant - 75
+      |  }
+      |}
+    """.stripMargin
```

Comment:
The Wdl in `ExecutionStoreSpec` and `SymbolStoreSpec` looks strikingly similar, can this be pulled out to `SampleWdl`?